src/domain/_common/components/Avatar/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { ReactComponent as AvatarIcon } from '@/assets/avatar-icon.svg';
+import generateRandomColor from '@/utils/generateRandomColor';
+
+interface AvatarProfileProps {
+  imageUrl?: string;
+  isSmall?: boolean;
```

Comment:
```suggestion
  size?: "small" | "large";
```
nit. 개인적인 취향인데... prop은 명사로 쓰는게 뭔가... 이뻐요... 🙂